ndk.rosinstall

```diff
@@ -222,70 +222,70 @@
-#     version: image_transport_plugins-release-release-kinetic-image_transport_plugins-1.9.5-0
+- tar:
+    local-name: image_common/camera_calibration_parsers
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/camera_calibration_parsers/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-camera_calibration_parsers-1.11.13-0
+- tar:
+    local-name: image_common/camera_info_manager
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/camera_info_manager/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-camera_info_manager-1.11.13-0
+- tar:
+    local-name: image_common/image_common
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/image_common/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-image_common-1.11.13-0
+- tar:
+    local-name: image_common/image_transport
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/image_transport/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-image_transport-1.11.13-0
+- tar:
+    local-name: image_common/polled_camera
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/polled_camera/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-polled_camera-1.11.13-0
+- tar:
+    local-name: image_pipeline/camera_calibration
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/camera_calibration/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-camera_calibration-1.12.23-0
+- tar:
+    local-name: image_pipeline/depth_image_proc
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/depth_image_proc/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-depth_image_proc-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_pipeline
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_pipeline/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_pipeline-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_proc
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_proc/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_proc-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_publisher
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_publisher/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_publisher-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_rotate
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_rotate/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_rotate-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_view
```

Comment:
There's an issue with `camera_calibration_parsers` here:

```
CMakeFiles/image_saver.dir/src/nodes/image_saver.cpp.o: In function `Callbacks::callbackWithCameraInfo(boost::shared_ptr<sensor_msgs::Image_<std::__ndk1::allocator<void> > const> const&, boost::shared_ptr<sensor_msgs::CameraInfo_<std::__ndk1::allocator<void> > const> const&)':
/opt/ros_android/output/catkin_ws/src/image_pipeline/image_view/src/nodes/image_saver.cpp:138: undefined reference to `camera_calibration_parsers::writeCalibration(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, sensor_msgs::CameraInfo_<std::__ndk1::allocator<void> > const&)'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [/opt/ros_android/output/catkin_ws/devel/image_view/lib/image_view/image_saver] Error 1
```

The patch is being applied correctly though. Does this happen to you as well?